### PR TITLE
feat(GridForm): added columnGap and rowGap as props to GridForm

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
## Added columnGap and rowGap as props to GridForm

We'll need to allow responsive layouts for our forms. This allows consumers of `GridForm` to specify their gaps.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.8.17-alpha.4db2ca.0<br/>@codecademy/gamut@3.0.1-alpha.4db2ca.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5e8f9bb381a5277fdaabff1f--codecademy-styleguide.netlify.com>https://5e8f9bb381a5277fdaabff1f--codecademy-styleguide.netlify.com</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5e8f9bb381a5277fdaabff1f>Deploy Logs</a></p>

File: packages/gamut/src/GridForm/index.tsx [hunk 1]
```diff
@@ -2,22 +2,32 @@ import React, { useEffect } from 'react';
...
 export function GridForm<
   Values extends Record<string, boolean | string | undefined>
->({ children, className, fields, submit, onSubmit }: GridFormProps<Values>) {
+>({
+  children,
+  className,
+  columnGap = 'lg',
+  fields,
+  onSubmit,
+  rowGap = 'md',
```
Comment (rogervera) [inline packages/gamut/src/GridForm/index.tsx:28]: Yay!
Since this is a visual change, could you write a story or add prop documentation for Storybook?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.8.17-alpha.5c9ba2.0<br/>@codecademy/gamut@3.0.1-alpha.5c9ba2.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5e90899a17210f7b0113ca20--codecademy-styleguide.netlify.com>https://5e90899a17210f7b0113ca20--codecademy-styleguide.netlify.com</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5e90899a17210f7b0113ca20>Deploy Logs</a></p>

**Pull request merged**